cmd/run/run.go

```diff
@@ -395,9 +395,11 @@ func (s *ServerContext) Run(ctx context.Context, config *serverconfig.Config) er
 
 	serverOpts = append(serverOpts, grpc.ChainUnaryInterceptor(
 		[]grpc.UnaryServerInterceptor{
-			requestid.NewUnaryInterceptor(),
+			grpc_ctxtags.UnaryServerInterceptor(),   // needed for logging
```

Comment:
can we coalesce these lines[1] into the statement that defines serverOpts for the first time[2] 

[1] https://github.com/openfga/openfga/blob/5ae864be30e386c4790d0520c12655bab3dd083e/cmd/run/run.go#L392-L412

[2] https://github.com/openfga/openfga/blob/5ae864be30e386c4790d0520c12655bab3dd083e/cmd/run/run.go#L392-L394